tools/api_proto_breaking_change_detector/detector_test.py

```diff
@@ -0,0 +1,103 @@
+from pathlib import Path
```

Comment:
It might be good to note that although these tests are already being run in buf's repo, they are here to specify the requirements from Envoy's point-of-view, and to ensure consistency.

Reply:
What would be the best way to document this? Would a python comment be fine?

Reply:
Yep a python comment would do.